scripts/deploy-kubefed.sh

```diff
@@ -86,6 +58,13 @@ EOF
   ${cmd}
 }
 
+function create-namespace {
```

Comment:
could we leverage the --create-namespace flag?

Reply:
Good catch! Bump Helm version 3.2+